### PR TITLE
Meal Map issue 20

### PR DESCRIPTION
**Pull request by Sajjadalgburi**
in this issue i just fixed the styling for homepage and embedded the search component and the results component together

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -12,10 +12,13 @@
   "dependencies": {
     "@apollo/client": "^3.7.15",
     "@stripe/stripe-js": "^1.54.0",
+    "axios": "^1.6.8",
+    "bootstrap": "^5.3.3",
```
Comment (Jason897571) [inline client/package.json:16]: Do we use bootstrap? it will have conflict with tailwind

File: client/src/main.jsx [hunk 2]
```diff
@@ -1,35 +1,32 @@
...
+import ReactDOM from 'react-dom/client'
+import { createBrowserRouter, RouterProvider } from 'react-router-dom'
 import './output.css'
+import 'bootstrap/dist/css/bootstrap.min.css'
 
-import App from './App.jsx';
-import Home from './pages/Home';
-import ErrorPage from './pages/ErrorPage';
-import ResultPage from './pages/ResultPage';
-import Donation from './pages/Donation';
```
Comment (Jason897571) [inline client/src/main.jsx:9]: donation page should not be deleted. Or do you integrated donation page somewhere?

**Review by Jason897571 — CHANGES_REQUESTED** [review]
Please see my comment

**Comment by Sajjadalgburi** [comment]
MADE A MISTAKE

**Pull request closed without merging**